.github/workflows/upgrade-dependencies.yaml

```diff
@@ -0,0 +1,14 @@
+name: Create Upgrade-Pull-Requests
```

Comment:
What is the purpose of this workflow? Is it about creating upgrade PRs? 

We are already using renovate since some time and I haven't seen such upgrade PRs for version of dependent components. 
If that's the goal of this workflow, I suggest removing it as we already use renovate for this purpose.

cc @oliver-goetz @rfranzke

Reply:
@ialidzhikov : you are correct. I added this workflow to migration-pullrequests for only a (small) subset of repositories (those that declare component-references). Will remove, as you suggested.